config/initializers/link_thumbnailer.rb

```diff
@@ -0,0 +1,79 @@
+# Use this hook to configure LinkThumbnailer bahaviors.
+LinkThumbnailer.configure do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.